source/nodejs/adaptivecards-site/package.json

```diff
@@ -4,12 +4,13 @@
 	"private": true,
 	"scripts": {
 		"clean": "hexo clean",
-		"build": "hexo generate",
+		"build": "hexo generate && npm run generate-sw",
```

Comment:
do you know why this failed?
https://adaptivecardsci.z5.web.core.windows.net/pr/7473/designer
then ctrl+shift+i to developer mode

![image](https://user-images.githubusercontent.com/6290692/171295022-07c15536-eba2-4ba5-bc75-c2c8e7fee608.png)

Reply:
@licanhua This error is outlined in the comment above: https://github.com/microsoft/AdaptiveCards/pull/7473#issuecomment-1140010232

I created a fix for it in another PR (https://github.com/microsoft/AdaptiveCards/pull/7484), but we aren't sure if we want to include it